.circleci/config.yml

```diff
@@ -798,6 +798,8 @@ jobs:
           test_targets: "other skip:other.test_native_link_error_message"
   test-browser-chrome:
     executor: bionic
+    environment:
+      EMTEST_LACKS_WEBGPU: "1"
```

Comment:
Does this mean that some tests that previously ran under chrome will no longer run?  Or are all such tests already ones that didn't actually test anything?

i.e. are there some webgpu browser tests that don't actually requires a real GPU to run?

Reply:
`test_webgpu_basic_rendering*` and `test_webgpu_required_limits` previously skipped themselves if a WebGPU adapter was not available. Now they fail in that case.

`test_webgpu_get_device*` do not require a GPU, it actually doesn't touch the WebGPU API at all, it only tests some Emscripten code.

Reply:
In that case I wonder if we should move `test_webgpu_get_device` to `test_other.py` so it doesn't depend on the browser?    Otherwise won't we loose that coverage?

Reply:
I didn't add `@requires_webgpu` to those tests, so they should still run.

If we moved them to `test_other.py` how would we run them, under Node?

Reply:
Sounds good!

(yes other tests run under either node or d8)